notebooks/04-transformers-text-classification.ipynb

```diff
@@ -1,5 +1,12 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Why is this included in this PR?

Reply:
That was a mistake, probably bc I rebased to master at some point (after dev1.2 was created). Fixed now